edf.md

```diff
@@ -0,0 +1,70 @@
+# Envelope data field
+
+> Version: 0.1.0 (Draft)
+> 
+> Authors: Oskar Thorén <oskar@status.im>, Dean Eigenmann <dean@status.im>
+
+## Table of Contents
+
+ 1. [Abstract](#abstract)
+ 2. [Specification](#specification)
+    1. [ABNF](#abnf)
+    2. [Signature](#signature)
+    3. [Padding](#padding)
+ 3. [Copyright](#copyright)
+
+## Abstract
+
+In this specification, we describe the [data field used in Waku](./waku.md#abnf-specification), this is relevant for decrypting incoming messages.
```

Comment:
might be worth reffing to eip627 as we took it from there

Reply:
Come to think if it, it is a bit of a weird sentence in the context of this separate spec.

Maybe change into:
```suggestion
In this specification, we describe the [data field used in Waku](./waku.md#abnf-specification), is it is used for the encryption and decryption of messages.
```

And for clarification, we could add something in the waku spec as:
Understanding the format of the data field is only required when posting new envelopes or decrypting incoming envelopes, not when a node is only forwarding envelopes.